app/controllers/settings/payments_controller.rb

```diff
@@ -54,6 +54,22 @@ def update
 
     return unless update_user_compliance_info
 
+    if params[:refund_card].present? && !current_seller.refund_credit_card.present?
```

Comment:
If the card is present, and the creator _has_ a credit card, how does one edit the card?

I think this code covers adding a card, and removing a card, but not updating the card.